lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -910,19 +911,22 @@ ThreadContext::IsNumericProperty(Js::PropertyId propertyId)
 const Js::PropertyRecord *
 ThreadContext::FindPropertyRecord(const char16 * propertyName, int propertyNameLength)
 {
-    Js::PropertyRecord const * propertyRecord = nullptr;
-
-    if (IsDirectPropertyName(propertyName, propertyNameLength))
+    if (propertyNameLength < 2)
```

Comment:
Consider adding a comment here to explain that we check for 2 here because empty string property records have 0 length, and propertyNamesDirect is a cache for single character property records